test.py

```diff
@@ -2,5 +2,5 @@
 
 x = random.randint()
 
-def poo (a=90, b=80):
-    return a + b
+def third (a = 90, b = 80):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -2,5 +2,6 @@
 
 x = random.randint()
 
-def third (a = 90, b = 80):
+
+def third(a=90, b=80):
     return a+b
```